### DISCUSSION TITLE
振り返りレポートが公開ページになってシェアできると嬉しい

### DISCUSSION THREAD

**Discussion by JunichiIto** [increments/qiita-discussions#403] [要望・アイデア・不具合]
振り返りレポートがメールで送られてきましたが、Twitter等でシェアしようと思ってもスクショを撮ってシェアするぐらいしかできません。
メールだけでなく、公開ページとしてブラウザでアクセスできるようになっていると気軽にURLをシェアできて嬉しいです。

ただ、レポートを見られたくない人もいると思うのでURLは推測しにくいURLにしておく必要があるかなと思います。

**Comment by tanaka-asami415** [comment]
@JunichiIto 
ご不便をおかけして申し訳ありません🙇
今年は今回いただいたようなことも、ぜひやりたいと思っております💪✨

**Comment by tanaka-asami415** [comment]
@JunichiIto 
今年の振り返りレポートをwebページとして公開いたしました！
https://x.com/Qiita/status/1739472935499493454?s=20

ただ申し訳ないことに、今年はまだページをシェアしていただくことができない状態です🙏

個人情報的な観点から、ユーザーさんご自身で「公開・非公開」を切り替えられる方がいいと判断したのですが
この機能を今年実装するのは工数的に厳しかったため、今回はWebページ化のみとなりました🙇
シェア機能に関しては来年ご用意できるようにする予定です！

**Reply by JunichiIto** [reply]
ご連絡ありがとうございます！わかりやすく情報がまとまっていてとてもよかったです 👍 
来年のアップデートにも期待しています〜 😄

**Reply by P-SiZK** [reply]
@JunichiIto 
お待たせしました 🎉 

ユーザー振り返りレポートを、ユーザーさんご自身で「公開する、公開しない」を切り替えられる外部公開機能を追加しました！

詳しい内容は[Qiita Blog](https://blog.qiita.com/yearly-summary-2024/)をご覧ください！

**Reply by JunichiIto** [reply]
@P-SiZK 確認しました。対応どうもありがとうございます！
華やかな感じで楽しいページですね 😄
のちほどシェアしようと思います！
